circle.yml

```diff
@@ -211,6 +205,27 @@ jobs:
           command: |
             ~/build/test/evmc-vmtester target/x86_64-unknown-linux-gnu/debug/libexamplerustvm.so
 
+  bindings-rust-msan:
+    docker:
+      - image: rust:1
+    steps:
+      - checkout
+      - run:
+          name: Update environment
+          command: |
+            echo 'deb http://apt.llvm.org/stretch/ llvm-toolchain-stretch-8 main' >> /etc/apt/sources.list
+            apt-key adv --keyserver keyserver.ubuntu.com --no-tty --recv-keys 6084F3CF814B57C1CF12EFD515CF4D18AF4F7421 60C317803A41BA51845E371A1E9377A2BA9EF27F
+            apt -qq update
+            apt -yq  install llvm-8-dev clang-8
+            rustup toolchain install nightly-x86_64-unknown-linux-gnu
```

Comment:
Why do you need to update rust if you use a docker image?

Reply:
This is about installing nightly, it is not available by default.

Reply:
Yes, I know. It would be good to make a custom image with LLVM and rust nightly or run different configuration builds as a sequence of steps, not independent jobs.